app/templates/travis.yml

```diff
@@ -1,15 +1,12 @@
 dist: trusty
 language: node_js
 node_js:
-  - 4
   - 6
   - node
```

Comment:
Doesn’t seem to be the latest version. 
Compare to https://github.com/micromata/bootstrap-kickstart/blob/v4-dev/.travis.yml